tests/codebuild/delete_tests.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+source common.sh
+source create_tests.sh
```

Comment:
It's just for that one instance of `verify_test`. That method is otherwise only ever used in `create_tests`, and it's really big. I just didn't want to move it out of that file ;(